### ISSUE TITLE
Don't pass mask

### ISSUE THREAD

**Issue by maksugr** [react-native-community/react-native-text-input-mask#20]
Can I use `TextInputMask` but with `undefined` or some other value of property mask (any character)? It's comfortable to use this component with mask and without.

**Comment by maksugr** [comment]
Anybody here? Can I somehow use mask for any symbol?)

**Comment by ivanzotov** [comment]
@maksugr yes, it checks if mask prop available – https://github.com/react-native-community/react-native-text-input-mask/blob/master/index.js#L23, so you can use `TextInputMask` like a `TextInput` without mask features. Let me know if it doesn't work :)

**Comment by kirakik** [comment]
@ivanzotov I get a crash in Android if I do not pass a mask
![screenshot_20171023-153753](https://user-images.githubusercontent.com/17203119/31909508-5b56d614-b808-11e7-8cf9-4dd5ceea225c.png)

**Comment by ivanzotov** [comment]
@kirakik thanks, reopened issue, will fix

**Comment by maksugr** [comment]
@ivanzotov yes, I had the same issue. sorry for non informative first message.

**Comment by kirakik** [comment]
@maksugr a work around for now would be to create a component that renders a text input, and pass it a `mask` prop when you need it. Then in the rendering of that component, you can do:

```js
const Input = this.props.mask ? TextInputMask : TextInput
return (
    <Input {...} />
)

**Comment by maksugr** [comment]
@kirakik yes, this way is good and work, but not for me. I also wrap components in styled-components and my logic is more complicated then this, and it doesn't work without additional hacks. So I want TextInputMask that works without mask prop by default.

**Comment by ivanzotov** [comment]
@maksugr fixed in `0.6.2`

**Comment by AnnaCarolinaWGC** [comment]
i still get the same non-null error....
